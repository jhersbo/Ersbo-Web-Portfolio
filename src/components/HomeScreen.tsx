import "../sass/homeScreen.scss"
import About from "./minor-components/About"
import ContactMe from "./minor-components/ContactMe";
import Projects from "./minor-components/Projects";

const HomeScreen = ()=>{
    
    return(
        <div id="container">
            <About/>
            <Projects/>
            <ContactMe/>
        </div>
    )
}

export default HomeScreen