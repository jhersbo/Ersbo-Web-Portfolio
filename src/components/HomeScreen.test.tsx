import HomeScreen from './HomeScreen'
import renderer from 'react-test-renderer'

test('passes snapshot test', () => {
  const tree = renderer.create(<HomeScreen/>).toJSON()
  expect(tree).toMatchSnapshot()
});