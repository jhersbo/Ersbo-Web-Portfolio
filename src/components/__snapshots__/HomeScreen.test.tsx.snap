// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passes snapshot test 1`] = `
<div
  id="container"
>
  <div
    className="box"
    style={Object {}}
  >
    <h3
      onClick={[Function]}
    >
      About Me
    </h3>
    <button
      onClick={[Function]}
    >
      Button
    </button>
  </div>
  <div
    className="box"
    style={Object {}}
  >
    <h3
      onClick={[Function]}
    >
      My Projects
    </h3>
    <button
      onClick={[Function]}
    >
      Button
    </button>
  </div>
</div>
`;
